packages/components/translations/en-us.yaml

```
@@ -0,0 +1,44 @@
+hds:
+  components:
+    shared:
+      loading: Loading
+      menu: Menu
+      optional: Optional
+      required: Required
+      sort-by: Sort by
+      toggle: Toggle
+      tooltip-prefix: More information for
+    advanced-table:
+      th-button-expand:
+        aria-description: Toggle the visibility of the related rows.
```

```suggestion
        aria-description: Toggle the visibility of the related rows
```
[Nit] Probably don't need punctuation